ci/tests/sonarqubetest/sonar_auth_test.go

```diff
@@ -0,0 +1,141 @@
+package main
+
+import (
+    "os"
+    "net/http"
+    "github.com/DATA-DOG/godog"
+    "io/ioutil"
+    "fmt"
+    "net/url"
+    "time"
+    "strings"
+)
+
+var sonarUrl string
+var body string
+var httpClient = &http.Client{
+      Timeout: time.Second * 10,
+}
+
+const (
+    SONAR_AUTH_ENDPOINT            string = "/api/authentication/login"
+    SONAR_GITHUB_AUTH_ENDPOINT     string = "/sessions/new?return_to=%2Fprojects"
```

Comment:
Does a direct project URL, such as https://sonar.buildstack-integration.build.finkit.io/projects, work here? So, you try to hit the project URL, but receive login content, not project content?

Reply:
That was introduced to test the anonymous user login access(Integration test). This would work here and isn't supposed to :)